README.md

```diff
@@ -11,13 +11,13 @@
 ![CommitActivity](https://img.shields.io/github/commit-activity/4w/apigee/apigeelint)
 ![Downloads](https://img.shields.io/npm/dm/apigeelint.svg)
 
-Static code analysis for Apigee proxy and sharedflow bundles to encourage API developers to use best practices and avoid anti-patterns.
+Static code analysis for Apigee proxy and sharedflow bundles to encourage API developers to use [best practices](https://docs.apigee.com/api-platform/fundamentals/best-practices-api-proxy-design-and-development) and avoid [anti-patterns](https://docs.apigee.com/api-platform/antipatterns/intro).
```

Comment:
Can we use these links instead - https://cloud.google.com/apigee/docs/api-platform/fundamentals/best-practices-api-proxy-design-and-development
https://cloud.google.com/apigee/docs/api-platform/antipatterns/intro

Reply:
Hi @ssvaidyanathan sure ... let me fix them! thank you very much